### PR TITLE
Refresh the CD part

### PR DESCRIPTION
**Pull request by jlevesy**
### What does this PR do?

This PR updates the lecture content to use google cloud as CD platform instead of Heroku

Still a draft, I'm still unhappy with some stuff, also I need to fix the accents 🙃.

Relates to https://github.com/cicd-lectures/slides/issues/89

### CODE REVIEW

File: content/chapitres/cd.adoc [hunk 1]
```diff
@@ -114,45 +114,30 @@ Source : http://blog.crisp.se/2013/02/05/yassalsundman/continuous-delivery-vs-co
...
 
-== Exercice: Créez une application sur Heroku (1/2)
+== Image de Container?
 
-* Rendez-vous sur link:https://heroku.com[Heroku] et créez vous un compte
-* Une fois sur la page link:https://dashboard.heroku.com/apps[Apps] cliquez sur `New app` dans le menu en haut à gauche
-* Sélectionnez un nom et une région (europe de préférence)
-* Cliquez sur Create App
+* Un container est une instance d'une application packagée
+* Un container est créé a partir d'une image, le disque d'installation de l'application
```
Comment (dduportal) [inline content/chapitres/cd.adoc]:
```suggestion
* Un container est:
** Une instance d'une application empaquetée
** Créé a partir d'une image (le disque d'installation de l'application)
```

nit(factorisation de texte)

File: content/chapitres/cd.adoc [hunk 2]
```diff
@@ -114,45 +114,30 @@ Source : http://blog.crisp.se/2013/02/05/yassalsundman/continuous-delivery-vs-co
...
-
-Heroku supporte plusieurs modes de déploiement
-
-* Push du code source dans un remote heroku via git
-* Connection directe avec GitHub
-* Push d'une image dans une container registry
-** C'est cette méthode que nous allons utiliser (car la moins auto-magique)
-
-== Construire une Image de Container pour Heroku
+* A la racine de votre dépôt menu-server créez un fichier `Dockerfile` avec le contenu suivant.
```
Comment (dduportal) [inline content/chapitres/cd.adoc]:
```suggestion
* A la racine de votre dépôt menu-server créez un fichier `Dockerfile` avec le contenu suivant :
```

nit(ponctuation)

**Review by dduportal — APPROVED** [review]
First review: it sounds really good 👍 

If it is ok for you, let's merge it when you feel ready, and then we improve by tiny increments

**Comment by jlevesy** [comment]
Sounds good, merge away

**Pull request merged**